tutorial/input_output.py

```diff
@@ -0,0 +1,234 @@
+from .common import Question, Quiz
+
+
+class StringOutput(Quiz):
+    def __init__(self, title=""):
+        q1 = Question(
+            question="Can the function <code>print()</code> be used on strings only?",
+            options={
+                "False": "Correct! It can be used on any Python object.",
+                "True:": "Wrong! Try it in a cell below.",
+            },
+            correct_answer="False",
+            shuffle=True,
+        )
+
+        q2 = Question(
+            question="What does a <code>f</code> before a string do? E.g <code>f'Hello {name}'</code>",
+            options={
+                "It formats the string as a float": "Wrong! Try it in a cell below.",
+                "It changes the color of the string": "Wrong! Try it in a cell below.",
+                "It allows you to insert variables into the string": "Correct! In the example above, it will print the value of the <code>name</code> variable, if this is defined.",
+            },
+            correct_answer="It allows you to insert variables into the string",
+            shuffle=True,
+        )
+
+        super().__init__(questions=[q1, q2])
+
+
+class StringInput(Quiz):
+    def __init__(self, title=""):
+        q1 = Question(
+            question="What does the function <code>input</code> do?",
+            options={
+                "It asks for user input and returns it as a string": "Correct! It can be used to read user input from the console.",
+                "It shows a list of input devices available on the current computer.": "Wrong! It is used to read user input from the console.",
+            },
+            correct_answer="It asks for user input and returns it as a string",
+            shuffle=True,
+        )
+
+        q2 = Question(
+            question="What happens if you call <code>input()</code> in the middle of a function?",
+            options={
+                "The function execution stops and it waits for the user to type an input": "Correct! Input is a blocking function which waits for user to enter a string in the console and press enter.",
+                "The function continues its execusion": "Wrong!",
+            },
+            correct_answer="The function execution stops and it waits for the user to type an input",
+            shuffle=True,
+        )
+
+        super().__init__(questions=[q1, q2])
+
+
+class Paths(Quiz):
+    def __init__(self, title=""):
+        q1 = Question(
+            question="What does the operator <code>/</code> do when applied to two <code>Pathlib.Path</code> objects?",
+            options={
+                "It removes the second path from the first": "Wrong, try it in the shell.",
+                "It concatenates paths.": "Correct, it lets you construct a path from different segments",
```

Comment:
```suggestion
                "It concatenates paths": "Correct, it lets you construct a path from different segments",
```

Otherwise it doesn't match with the answer